README.md

```diff
@@ -18,14 +18,13 @@ Duffle is a command line tool that allows you to install and manage CNAB bundles
     $ cd duffle
     ```
 
-3. Vendor dependencies and build the `duffle` binary:
+3. Vendor dependencies, build the `duffle` binary and install onto your system:
     ```console
-    $ make bootstrap build
+    $ make bootstrap build install
```

Comment:
`make install` needs sudo permissions on most computers since by default it installs into /usr/local/bin:

```
$ make bootstrap build
$ sudo make install
```

Reply:
👍 updated